src/handlers/handleAudio.ts

```diff
@@ -148,8 +154,8 @@ async function sendTranscription(ctx: Context, url: string, fileId: string) {
           await ctx.dbchat.save()
         }
       }
-    } catch (error) {
-      report(error, { ctx, location: 'removeBadWitToken' })
+    } catch (err) {
```

Comment:
Don't shorten the error variable name

Reply:
I shorten them cause they were already defined in an upper scope, on a catch block above the current block. Rolling back now.